src/Action/AddAction.php

```diff
@@ -126,7 +126,7 @@ protected function _post()
             return $this->_success($subject);
         }
 
-        return $this->_error($subject);
+        $this->_error($subject);
```

Comment:
We can change return type of `_erro()` to `Reponse|null` and add back this `return`.  That should make phpstan happy.